spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/client/GrpcChannelFactoryConfigurations.java

```diff
@@ -72,7 +71,8 @@ ShadedNettyGrpcChannelFactory shadedNettyGrpcChannelFactory(GrpcClientProperties
 	}
 
 	@Configuration(proxyBeanMethods = false)
-	@ConditionalOnClass(value = { Channel.class, NettyChannelBuilder.class })
+	@ConditionalOnClass({ io.grpc.netty.shaded.io.netty.channel.Channel.class,
```

Comment:
This config is for the non-shaded NettyChannelFactory so this should stay non-fully-qualified and just add the import back in.